frontend/src/App.js

```diff
@@ -1,34 +1,7 @@
-import React from "react";
+import React from 'react';
 
-class TestComponent extends React.Component {
-    constructor(props) {
-        super(props);
-        this.state = {
-            gameCreated: false
-        }
-    }
+import GameList from 'components/GameList';
 
-    componentDidMount() {
-        fetch("api/createGame", {headers:{"accepts":"application/json"}})
-            .then(
-                (result) => {
-                    console.log(result)
-                    this.setState({
-                        gameCreated: true
-                    })
-                },
-                (error) => {}
-            )
-    }
+const App = () => <GameList />;
```

Comment:
Nice! Not very clear on how we expand from here.

My initial guess is [conditional rendering](https://reactjs.org/docs/conditional-rendering.html). Would that be inline in the `App` or have another abstraction?

Might be good to draw that out somehow. Either way though - def out of the scope of this pr 👍 !

Reply:
Might introduce another abstraction when I get around to building another "page" :)